review_practice/react-app/src/App.js

```diff
@@ -0,0 +1,32 @@
+// Import React modules
+import React from 'react';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+
+import { ApolloProvider } from '@apollo/react-hooks';
+
+// Import pages
+import { SearchRoom } from 'pages/SearchRoom';
+import Login from 'pages/Login';
+import RegisterUser from 'pages/RegisterUser';
+
+// Import components
+import GlobalStyle from 'components/GlobalStyle';
+import Client from 'service/NetworkClient';
+
+// App
+const App = () => {
+    return (
+        <ApolloProvider client={Client}>
+            <Router>
+                <GlobalStyle />
+                <Switch>
+                    <Route exact path="/" component={Login} />
+                    <Route path="/user/register" component={RegisterUser} />
+                    <Route path="/main" component={SearchRoom} />
```

Comment:
`path` 같은 경우는 다른 컴포넌트에서 왕왕 쓰이므로 상수로 관리하는 것을 추천합니다.

```suggestion
                    <Route exact path={ROUTES.LOGIN} component={Login} />
                    <Route path={ROUTES.USER_REGISTER} component={RegisterUser} />
                    <Route path={ROUTES.MAIN} component={SearchRoom} />
```